apps/dashboard/src/main/java/com/akto/action/ProfileAction.java

```diff
@@ -124,6 +124,17 @@ public static void executeMeta1(Utility utility, User user, HttpServletRequest r
         } catch (Exception e) {
         }
 
+        long countUser = UsersDao.instance.count(Filters.and(
+                Filters.exists(User.NAME_LAST_UPDATE),
+                Filters.in("login", username)
+        ));
+
+        long countOrg = OrganizationsDao.instance.count(Filters.and(
```

Comment:
handle for which account we are storing org, there is no account id check, what if i have 3 orgs